kernel/console/kprintf.c

```diff
@@ -45,11 +44,11 @@ uint64_t kprintf(const char *s, ...){
 		
 		idx_in++;
 		
-		if (idx_out == 63){
-			out_str[63] = '\0';
+		if (idx_out == KPRINTF_LEN){
+			out_str[KPRINTF_LEN] = '\0';
```

Comment:
I think this will be a buffer overrun--`out_str[KPRINTF_LEN]` won't actually exist, no?